edge/pkg/edged/edged.go

```diff
@@ -338,7 +342,7 @@ func (e *edged) Start() {
 
 	housekeepingTicker := time.NewTicker(housekeepingPeriod)
 	syncWorkQueueCh := time.NewTicker(syncWorkQueuePeriod)
-	e.probeManager.Start()
```

Comment:
Since the probeManager.Start() was deleted in this version of k8s, we need to check whether the probe feature for pod is worked correctly. :)

Reply:
by test, liveness probe works well.